Vagrantfile

```diff
@@ -23,7 +23,8 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     # Execute commands in the frontend directory
     fe.exec.commands %w(rails rake rspec bundle), directory: '/vagrant/src/api'
     fe.exec.commands 'script/start_test_backend', directory: '/vagrant/src/api'
-    fe.exec.commands 'contrib/start_development_backend', directory: '/vagrant'
+    fe.exec.commands 'start_development_backend', directory: '/vagrant/contrib/'
```

Comment:
I switched back to `contrib/start_development_backend` on purpose so it's looks more like `script/start_test_backend`